inc/saitypes.h

```diff
@@ -441,6 +441,27 @@ typedef struct _sai_prbs_rx_state_t
     uint32_t error_count;
 } sai_prbs_rx_state_t;
 
+typedef struct _sai_bool_current_and_changed_status_t
+{
+    /** Current status at the time of read */
+    bool current_status;
+
+    /** Indicates that the status changed at least once since the last read */
+    bool changed;
+} sai_bool_current_and_changed_status_t;
+
+typedef struct _sai_port_lane_bool_current_and_changed_status_t
+{
+    uint32_t lane;
+    sai_bool_current_and_changed_status_t value;
+} sai_port_lane_bool_current_and_changed_status_t;
```

Comment:
should this struct have more meaningful name rather than names and types of fields?

Reply:
I changed the name of the struct. Please check the new revision